task_01_iryna/src/Algorithms/BirthdayAlgorithm.java

```diff
@@ -1,20 +1,18 @@
 package Algorithms;
 
-import Algorithms.Algorithm;
-
-public class BirthdayAlgorithm extends Algorithm {
+public class BirthdayAlgorithm implements Algorithm {
     private static final int DAYS_IN_A_YEAR = 366;
     private static final int NUMBER_OF_PEOPLE = 23;
 
     public double getResult() {
-        double k = 1.0;
+        double compositionOfProb = 1.0;
 
         for (int i = DAYS_IN_A_YEAR - NUMBER_OF_PEOPLE; i <= DAYS_IN_A_YEAR - 1; i++) {
-            k = k * i;
-            k = k / 365;
+            compositionOfProb = compositionOfProb * i;
+            compositionOfProb = compositionOfProb / 365;
```

Comment:
What about `365`? You can extract it to another constant field or compute it from `DAYS_IN_A_YEAR`